vms/proposervm/vm_test.go

```diff
@@ -2000,6 +1995,8 @@ func TestVM_VerifyBlockWithContext(t *testing.T) {
 
 	// Create a VM
 	innerVM := blockmock.NewChainVM(ctrl)
+	innerVM.EXPECT().WaitForEvent(gomock.Any()).Return(common.Message(0), nil).AnyTimes()
```

Comment:
Should we be returning `common.PendingTxs` here? `0, nil` is really never valid to be returned.